bentoctl/docker_utils.py

```diff
@@ -51,9 +52,13 @@ def __rich_console__(self, *_):
 
 
 def generate_deployable_container(
-    tag: str, deployment_config: DeploymentConfig, cleanup: bool
+    tag: str, deployment_config: DeploymentConfig
 ) -> None:
-    with TempDirectory(cleanup=cleanup) as dist_dir:
+    """
+    Calls the operator and generates the deployable. If in debug more the generated
+    deployable will be moved to the current directory for easier debugging.
+    """
+    with TempDirectory(debug=get_debug_mode()) as dist_dir:
```

Comment:
Moving directory is a bit unnatural for debugging. Maybe preserving and printing the temp directory during debug should be sufficient?